alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -761,4 +760,21 @@ protected function getBestMatchFlavor($groupedFlavors, $referenceFlavor)
 		return $flavor;
 	}
 
+	/**
+	 * @param string $path
+	 * @param int $flavorId
+	 * @return array
+	 */
+	private function getClipData($path, $flavorId)
+	{
+		$mediaInfo = mediaInfoPeer::retrieveByFlavorAssetId($flavorId);
+		$clipDesc = array('type' => 'source', 'path' => $path);
+		if ($mediaInfo && !$mediaInfo->getAudioBitRate())
+		{
```

Comment:
Please add a debug info here, so we can cover bizarre cases where there is audio track but not bit rate is set